sbin/pfmon

```diff
@@ -408,18 +409,24 @@ sub _runtask {
             pause;
```

Comment:
There is no need to reset it since it is not used until it is set to the new value, and it will be set unconditionally